src/main/resources/db/migration/V4_CreateProject.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE project (
+    projectId INT NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(50) NOT NULL,
+    `value` DECIMAL(11, 2) NOT NULL,
+    techLeadId INT NOT NULL,
```

Comment:
looks like this is supposed to be a fkey that links to a techLead lkp table? Is this in scope here or coming in another ticket?

Reply:
see line 12